Detectors/TPC/reconstruction/run/testRawReader.cxx

```diff
@@ -0,0 +1,95 @@
+///
+/// @file   testRawReader.cxx
```

Comment:
yes, it is a development tool. where should we place such tool in the future to avoid confusion?